test/e2e/clusterctl_upgrade_test.go

```diff
@@ -84,8 +84,8 @@ var _ = Describe("When testing clusterctl upgrades (v1.4=>current)", func() {
 			InitWithInfrastructureProviders: []string{"docker:v1.4.5"},
 			InitWithProvidersContract:       "v1beta1",
 			// NOTE: If this version is changed here the image and SHA must also be updated in all DockerMachineTemplates in `test/e2e/data/infrastructure-docker/v1.4/bases.
```

Comment:
CAPI v1.4 does not and will not support 1.28. Let's stay on 1.27 for both v1.4 tests